build_docker_remote.sh

```diff
@@ -0,0 +1,103 @@
+#!/usr/bin/env bash
+#
+# Build (and optionally push) a GATK docker image.
+#
+# If you are pushing an image to our release repositories, be sure that you've followed
+# the setup instructions here:
+# https://github.com/broadinstitute/gatk/wiki/How-to-release-GATK4#setup_docker
+# and here:
+# https://github.com/broadinstitute/gatk/wiki/How-to-release-GATK4#setup_gcloud
+#
+
+# Have script stop if there is an error
+set -e
+
+REPO=broadinstitute
+PROJECT=gatk
+REPO_PRJ=${REPO}/${PROJECT}
+GCR_REPO="us.gcr.io/broad-dsde-methods/broad-gatk-snapshots/gatk-remote-builds"
+STAGING_CLONE_DIR=${PROJECT}_staging_temp
+
+#################################################
+# Parsing arguments
+#################################################
+while getopts "e:pslrud:t:" option; do
+	case "$option" in
+		e) GITHUB_TAG="$OPTARG" ;;
+		s) IS_HASH=true ;;
+		d) STAGING_DIR="$OPTARG" ;;
+		t) DOCKER_IMAGE_TAG="$OPTARG" ;;
+	esac
+done
+
+if [ -z "$GITHUB_TAG" ]; then
+	printf "Option -e requires an argument.\n \
+Usage: %s: -e <GITHUB_TAG> [-psl] \n \
+where <GITHUB_TAG> is the github tag (or hash when -s is used) to use in building the docker image\n \
+(e.g. bash build_docker.sh -e 1.0.0.0-alpha1.2.1)\n \
+Optional arguments:  \n \
+-s \t The GITHUB_TAG (-e parameter) is actually a github hash, not tag.  git hashes cannot be pushed as latest, so -l is implied.  \n \
+-l \t Do not also push the image to the 'latest' tag. \n \
```

Comment:
Shouldn't the default here be to not push to the latest tag? You'd only want to push to latest if it's an actual release right (or is that the default use case here)?